### PR TITLE
Fix typos and minor style problems

### PR DESCRIPTION
**Pull request by zmbc**
I'm a compulsive proofreader :laughing: , and we're reviewing this at the beginning of INFO 370.

- I only made changes I thought would be uncontroversial, but there's a bit of a gray area. For instance, there were a fair number of spots where spaces were used around arguments (e.g. `return( min(a) )`), but it seems to me that the Google Style Guide [says not to do that](https://google.github.io/styleguide/Rguide.xml#spacing) ("Do not place spaces around code in parentheses or square brackets.").
- There were a few things that seemed off to me but the solution would be more involved; would opening a GitHub issue be the right venue for that?
- Technical note: I think I generated the files correctly, but I got a bunch of [image diffs](https://github.com/zmbc/book/pull/1/files) that look to have to do with minor font changes, etc. Not sure what the root cause of that is but I have excluded them from this PR. Maybe double-check that the images aren't broken on your end.

Open to feedback/correction on everything; I'm enabling edits from maintainers so you can tweak things yourself if you want.

### CODE REVIEW

File: vectors.Rmd [hunk 1]
```diff
@@ -28,7 +28,7 @@ people.length <- length(people)
 print(people.length)  # [1] 3
 
 numbers <- c(1,2,3,4,5)
-print( length(numbers) )  # [1] 5
+print(length(numbers))  # [1] 5
```
Comment (joelwross) [inline vectors.Rmd]: The space around the inner expression is provided for clarity (to "group" the operations), which is useful for people who don't have practice reading nested expressions.

File: git-basics.Rmd [hunk 2]
```diff
@@ -46,14 +46,14 @@ Git was made to support completely decentralized development, where developers p
 
 Teams can setup their own servers to host these centralized repositories, but many choose to use a server maintained by someone else. The most popular of these in the open-source world is [GitHub](https://github.com/). In addition to hosting centralized repositories, GitHub also offers other team development features, such as issue tracking, wiki pages, and notifications. Public repositories on GitHub are free, but you have to pay for private ones.
 
-In short: GitHub is a site that provides as a central authority (or clearing-house) for multiple people collaborating with git. Git is what you use to do version control; Github is one possible place where repositories of code can be stored.
+In short: GitHub is a site that provides as a central authority (or clearing-house) for multiple people collaborating with git. Git is what you use to do version control; GithHb is one possible place where repositories of code can be stored.
```
Comment (joelwross) [inline git-basics.Rmd]: GitHb is further typo

File: functions.Rmd [hunk 3]
```diff
@@ -138,7 +138,7 @@ OTHERWISE
 In R, we write these conditional statements using the keywords **`if`** and **`else`** and the following syntax:
 
 ```r
-if(condition){
+if (condition) {
```
Comment (joelwross) [inline functions.Rmd]: I prefer no spaces between an `if` and its expression (a la Java), but will defer to style guide

**Comment by joelwross** [comment]
And yes, please feel free to open an issue for any more significant changes (or you can check in with us off-line).

File: vectors.Rmd (hunk 1 above)
Comment (zmbc) [reply vectors.Rmd]: Perhaps the best way is to restore the spacing but make it clearer that it's for educational purposes and isn't considered good style?

File: functions.Rmd (hunk 3 above)
Comment (zmbc) [reply functions.Rmd]: [Relevant quote](https://google.github.io/styleguide/Rguide.xml#spacing):
> Place a space before left parenthesis, except in a function call.
> GOOD: 
> `if (debug)`
> BAD: 
> `if(debug)`
